softvis3d-frontend/checkTestFileLocationAndName.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
I'm fine with doing it later... But it should be done, not later than the moment we want to integrate this test into our build process, so I can continue to build it on a Windows Machine as well ;-)